airbyte-server/src/main/java/io/airbyte/server/converters/JsonSecretsProcessor.java

```diff
@@ -22,12 +22,13 @@
  * SOFTWARE.
  */
 
-package io.airbyte.commons.json;
+package io.airbyte.server.converters;
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Preconditions;
+import io.airbyte.commons.json.Jsons;
 
 public class JsonSecretsProcessor {
```

Comment:
this protocol is pretty specific to the server. think it makes sense to live in that package.